exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestInfoSchema.java

```diff
@@ -220,7 +239,29 @@ public void describeWhenSameTableNameExistsInMultipleSchemas() throws Exception{
           .baselineValues("TABLE_TYPE", "CHARACTER VARYING", "NO")
           .go();
     } finally {
-      test("DROP VIEW dfs.tmp.`TABLES`");
+      test("DROP VIEW IF EXISTS dfs.tmp.`TABLES`");
+    }
+  }
+
+  @Test
+  public void describeWhenSameTableNameExistsInMultipleSchemasWithTableKeyword() throws Exception{
+    try {
+      test("USE dfs.tmp");
+      test("CREATE OR REPLACE VIEW `TABLES` AS SELECT full_name FROM cp.`employee.json`");
```

Comment:
Use select from values instead of cp (ex: `select 'A' as c from (values(1))`), this would load only one row which is faster than all rows from `employee.json`.

Reply:
Just realized that it's a view creation. You can leave it as is.